File: src/main/java/fr/lewon/web/bot/entities/girls/GirlQuests.java
package fr.lewon.web.bot.entities.girls;

import com.fasterxml.jackson.annotation.JsonProperty;

import fr.lewon.web.bot.entities.quests.QuestPremise;

public class GirlQuests {

	@JsonProperty("for_upgrade")
	private QuestPremise forUpgrade;

	public QuestPremise getForUpgrade() {
		return forUpgrade;
	}

	public void setForUpgrade(QuestPremise forUpgrade) {
		this.forUpgrade = forUpgrade;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/girls/GirlQuests.kt
package fr.lewon.bot.hh.entities.girls

import com.fasterxml.jackson.annotation.JsonProperty
import fr.lewon.bot.hh.entities.quests.QuestPremise

class GirlQuests {
    @JsonProperty("for_upgrade")
    var forUpgrade: QuestPremise? = null

}